modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -205,13 +205,17 @@ TEST_P(Test_ONNX_layers, Convolution_variable_weight_bias)
 
 TEST_P(Test_ONNX_layers, Gather)
 {
-    if (backend == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019 && target == DNN_TARGET_MYRIAD)
-        applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_MYRIAD, CV_TEST_TAG_DNN_SKIP_IE_NN_BUILDER);
```

Comment:
Removal of 'skip" checks means that this test passes.
It is not true, so we should kept existed conditions.

Don't change existed, add new only.

Reply:
This PR replaces old implementation and doesn't support anything except for CPU for now. Subset of Gather used to be implemented via Slice + Reshape, which support multiple backends.